README.md

```diff
@@ -24,6 +24,24 @@ Add the following to your Podfile:
 pod "react-native-heap", path: "../node_modules/@heap/react-native-heap"
 ```
 
+If your Podfile does not specify the path to your local React pod, you should add that (and any other necessary subspecs) to your Podfile. For example:
```

Comment:
Was this drawn from the docs page at the link on line 19?  If so, we should probably put them closer together, or possibly just direct the user to that page more strongly.

One other idea - maybe we could have a `Build Troubleshooting` section of the docs where we include the error that people saw and how that might fix it.  Thoughts?

Reply:
I pulled it from a Podfile I was working with, so not sure what docs page you're referring to, but would be happy to link to one. Can you share the link you're suggesting?

But yes, a `Build Troubleshooting` section sounds like a better idea than putting this directly in there. Share the link, and I'll handle both updates.

Reply:
This is the link from the RN docs: https://facebook.github.io/react-native/docs/integration-with-existing-apps#configuring-cocoapods-dependencies